runtime/permissions.rs

```diff
@@ -63,515 +65,506 @@ impl Default for PermissionState {
 
 #[derive(Clone, Debug, Default, Deserialize, PartialEq)]
 pub struct UnaryPermission<T: Eq + Hash> {
+  #[serde(skip)]
+  pub name: &'static str,
+  #[serde(skip)]
+  pub description: &'static str,
```

Comment:
Why serde skip? Is the Deserialize trait even used?

Reply:
yes, for worker permissions